packages/base/src/widget.ts

```diff
@@ -947,7 +955,9 @@ export class DOMWidgetView extends WidgetView {
   /**
    * Public constructor
    */
-  initialize(parameters: WidgetView.IInitializeParameters): void {
+  async initialize(
+    parameters: WidgetView.IInitializeParameters
+  ): Promise<void> {
```

Comment:
Doesn't this change the behaviour, previously `initialize` was not a promise.
In the past no one would wait for `initialze` to complete even though there were some internal promises, and now we're changing the flow of the code be making `initialize` an async function.
From my understanding this should still remain a sync function 
@jasongrout @ibdafna /cc

Reply:
Leaving this one a sync function